.github/workflows/publish-conda.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish Python 🐍 distributions 📦 to Conda
+
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to Conda
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: publish-to-conda
+      uses: blanderbuss/conda-publish-action@v1.4
+      with:
+        subDir: 'conda'
+        AnacondaToken: ${{ secrets.ANACONDA_TOKEN }}
+        platforms: 'linux'
```

Comment:
Is there a prepublish test we can do? Ex: In pip, we do a test publish to `test.pypi.org` first

Reply:
After researching, I don't belive that there is a way to perform a test publish for conda